app/controllers/imports_controller.rb

```diff
@@ -57,7 +57,7 @@ def update
       end
 
       if @import.update(import_params)
-        ImportWorker.perform_async(@import.id) if params[:commit] == "Process"
+        ImportWorker.perform_async(@import.id) if params[:commit] == "Process Update"
```

Comment:
So the intention behind this line and https://github.com/RepairShopr/importr/blob/6ec55ebec91d41bec11869557349857bb7fca44a/app/views/imports/show.html.erb#L80-L88 and https://github.com/RepairShopr/importr/blob/4c814163632ca3dd9414e02821d162f3a596c51f/app/assets/javascripts/import.js#L20-L23

Was to re-use the same web route to perform an Update with-running-worker and without-running-worker.

When editing `platform`, `subdomain`, `api_key` (in `show.html.erb`) and perhaps certain rows from `import.js`,
  the Import record in the `importr` database would be updated with those values *without* running an ImportWorker. It tries to do this by modifying `commit` to `Process Update` and then back.

For some reason, the `importr` app is no longer changing `Process Update` back to `Process`. or more precisely: it looks to be changing Process => Process Update => Process => and then back to Process Update

a part that looks suspicious is using `trigger("click.rails")` which is a jquery-ujs thing when the app is using Rails 6. back in Rails 5, they replaced the default/standard jquery-ujs with rails-ujs.  jquery-ujs seems to be supporting Rails 6, and working from production javascript console, but maybe there's a bug around that

Reply:
Thanks for this info, Steven! This is helpful.